index.rst

```diff
@@ -1,10 +1,13 @@
 Geophysics for Practicing Geoscientists
 =======================================
 
+<<<<<<< HEAD
+=======
```

Comment:
the easiest way to sort through this is to first try merging master onto your branch and then resolve the merge conflicts using sourceTree. (We can meet on tues if that is easiest!)